README.md

```diff
@@ -0,0 +1,70 @@
+# nri-plugins-operator
+
+The nri-plugins-operator is an Ansible-based operator created with operator-sdk to manage the life cycle of the
+nri-plugins. The operator deploys community maintained [nri-plugins](https://github.com/containers/nri-plugins) in
+Kubernetes cluster. When operator is installed, it doesn't do anything apart from watching for custom resources called
+NriPluginDeployment. When NriPluginDeployment object is created, reconciliation loops kicks off and installs the
+nri-plugin specified in the NriPluginDeployment. 
+
+## nriPluginDeployment object
+
+```YAML
+apiVersion: config.nri/v1alpha1
+kind: NriPluginDeployment
+metadata:
+  name: nriplugindeployment-sample
+  namespace: kube-system
+spec:
+  pluginName: topology-aware
+  pluginVersion: "v0.2.3"
+  state: present
+  values:
+    nri:
+      patchRuntimeConfig: true
+    tolerations:
+      - key: "node-role.kubernetes.io/control-plane"
+        operator: "Exists"
+        effect: "NoSchedule"
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: topology.kubernetes.io/disk
```

Comment:
Can we come up with an example of a node selection criteria that where the topology aware policy would be more obviously great choice. For instance: there is more than one NUMA node, or there are more than 3 CPU cores, or something like that.

If that is not easy to formulate with commonly available tools, then let's forget this. I'm anyway happy about the shape of this yaml.

Reply:
I think that's a good idea but I don't have immediate idea to be honest how to depict it here. Perhaps I will take care of it in the next iteration as a future improvement and for now we maybe focus on getting the basic operator in place.

Reply:
Yep, that's fine @fmuyassarov!